### PR TITLE
[#5419] Opening non-existent data object for read does not produce a stacktrace (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**
This change keeps the server log quiet.

### CODE REVIEW

File: scripts/irods/test/test_istream.py [hunk 1]
```diff
@@ -173,6 +174,18 @@ def test_client_side_configured_default_resource_is_honored_on_writes__issue_529
             self.admin.assert_icommand(['iadmin', 'rmresc', resc_name])
             self.user.environment_file_contents = old_env_config
 
+    def test_opening_a_non_existent_data_object_does_not_cause_a_stacktrace_to_be_logged__issue_5419(self):
+        # Capture the current log file size. This will serve as an offset for when we inspect
+        # the log file for messages.
+        log_offset = lib.get_file_size_by_path(paths.server_log_path())
+
+        # The following causes a stacktrace to be logged in iRODS v4.2.8 because the data object does not exist.
+        self.admin.assert_icommand(['istream', 'read', 'does_not_exit'], 'STDERR', ['Error: Cannot open data object.'])
```
Comment (trel) [inline scripts/irods/test/test_istream.py:183]: does_not_exit -> does_not_exist

Comment (korydraughn) [reply scripts/irods/test/test_istream.py]: done

File: scripts/irods/test/test_istream.py [hunk 2]
```diff
@@ -173,6 +174,18 @@ def test_client_side_configured_default_resource_is_honored_on_writes__issue_529
             self.admin.assert_icommand(['iadmin', 'rmresc', resc_name])
             self.user.environment_file_contents = old_env_config
 
+    def test_opening_a_non_existent_data_object_does_not_cause_a_stacktrace_to_be_logged__issue_5419(self):
```
Comment (alanking) [inline scripts/irods/test/test_istream.py:177]: This will need to be skipped in topology tests due to reading from the log, I think.

**Review by alanking — COMMENTED** [review]
Looks fine to me overall

File: scripts/irods/test/test_istream.py (hunk 2 above)
Comment (korydraughn) [reply scripts/irods/test/test_istream.py:179]: added skip decorator.

**Comment by trel** [comment]
green, please # it.

**Comment by korydraughn** [comment]
passed CI.

added #

**Comment by korydraughn** [comment]
please hold on merging this. i want to add one more check to the test.

**Comment by korydraughn** [comment]
updated and verified that change to test passed.

this can be merged now.

**Pull request merged**